README.md

```diff
@@ -35,7 +57,47 @@ Once the Containerfile is generated, you can build the image using either Podman
 ### Using Podman build image for x86_64
 
 ```bash
-podman build --platform linux/amd64 -f distribution/Containerfile -t rh .
+podman build --platform linux/amd64 -f distribution/Containerfile -t llama-stack-rh .
+```
+
+### Using Docker
+
+```bash
+docker build -f distribution/Containerfile -t llama-stack-rh .
+```
```

Comment:
It would make more sense to unified these sections - we don't need separate ones for basically the same command. I've noticed the Docker command has no arg for the platform arch?